app/services/messages_services.py

```diff
@@ -0,0 +1,64 @@
+from http import HTTPStatus
```

Comment:
The model and controller names are singular, the services should be singular too, `message_services`. I'll approve anyway.